src/settings/OpeningPeriodForm/BigCalendarWrapper.js

```diff
@@ -53,9 +53,16 @@ class BigCalendarWrapper extends PureComponent {
         }
       }) => {
         const event = {};
+        // localized start of week
```

Comment:
Could you please try to make names of constants more descriptive, so that we don't need comments. Here and below. Thanks!